service/controller/chart/v1/resource/release/desired.go

```diff
@@ -27,7 +28,12 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 
 	tarballPath, err := r.helmClient.PullChartTarball(ctx, tarballURL)
 	if err != nil {
-		return nil, microerror.Mask(err)
+		r.logger.LogCtx(ctx, "level", "error", "message", "pulling chart failed", "stack", microerror.Stack(err))
```

Comment:
Putting a stack in the end since network error could vary.

Reply:
When we expect an error the logging should only be `warning`.